apps/postybirb-ui/src/hooks/account/use-websites.ts

```diff
@@ -51,7 +51,7 @@ export function useWebsites() {
   );
 
   const filteredWebsites = websites.filter(
-    (website) => !settings.hiddenWebsites.includes(website.id),
+    (website) => !settings?.hiddenWebsites.includes(website.id),
```

Comment:
It was throwing errors for some reason.

Reply:
I think it is some kind of race condition or somesuch. I've had it randomly throw before as well.